### ISSUE TITLE
First Tribler 8.04 test conversion old library

### ISSUE THREAD

**Issue by Nubro01** [Tribler/tribler#8271]
Hi! I was using Tribler and THIS happened! :cry:
```
Traceback (most recent call last):
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\asyncio\events.py", line 80, in _run
  File "D:\a\tribler\tribler\pyipv8\ipv8\taskmanager.py", line 153, in done_cb
  File "D:\a\tribler\tribler\pyipv8\ipv8\taskmanager.py", line 28, in interval_runner
  File "D:\a\tribler\tribler\src\tribler\core\torrent_checker\torrent_checker.py", line 280, in check_local_torrents
  File "D:\a\tribler\tribler\src\tribler\core\torrent_checker\torrent_checker.py", line 280, in <listcomp>
  File "D:\a\tribler\tribler\src\tribler\core\torrent_checker\torrent_checker.py", line 343, in check_torrent_health
  File "D:\a\tribler\tribler\src\tribler\core\torrent_checker\torrent_checker.py", line 371, in create_session_for_request
TypeError: '>' not supported between instances of 'str' and 'int'

rE@http://127.0.0.1:51452/ui/assets/index.9ee516a5.js:80:336

```

**Comment by qstokkink** [comment]
Thanks for reporting! This seems similar to #8270. However, in this case, the key in your configuration file (%APPDATA%/.Tribler/8.0/configuration.json) that got corrupted was `libtorrent` / `download_defaults` / `number_hops`.

I'll rework the workaround from #8270 for your case. You have the following workarounds available:

 a. Remove the quotes around the value of `number_hops`. For example, `1` instead of `"1"`.
 b. Remove the entire `configuration.json` to reset to default settings.

Thanks to your comment that this happened after conversion, I think I know how to fix this now.

**Comment by qstokkink** [comment]
This will be fixed in the upcoming 8.0.5 version. In the next version, you can remove the `.upgraded` and `configuration.json` files from your `.Tribler/8.0` directory and importing your old settings should work correctly.
